dashboard/db/migrate/20190422234557_add_course_to_pd_scholarship_info.rb

```diff
@@ -0,0 +1,7 @@
+class AddCourseToPdScholarshipInfo < ActiveRecord::Migration[5.0]
+  def change
+    add_column :pd_scholarship_infos, :course, :string
+
+    add_index :pd_scholarship_infos, [:user_id, :application_year, :course], unique: true, name: 'index_pd_scholarship_infos_on_user_id_and_app_year_and_course'
```

Comment:
I don't have a good sense of when to add an index like this.. what's the philosophy here?

Reply:
I added this index because This combination of values is how we will be looking up an individual `ScholarshipInfo`, and because we need this combination to be unique (a user can only have one scholarship per course per year).

Reply:
Oh, that second reason is very interesting.  Thanks.